### DISCUSSION TITLE
Feature request: Add actions to queues paused

### DISCUSSION THREAD

**Discussion by Ytrog** [doublecmd/doublecmd#1142] [Suggestions]
Often when I click "add to queue" for example when moving I want an option to add it in a paused state so I can control when it starts. Now the whole queue feature seems useless as it starts immediately and finishes just as fast. I would use it to plan things ahead.

**Comment by tomas-chrastina** [comment]
+1. 

I've had exactly the same idea.

It's rather problematic, when we want copy few files from different locations over network.
If copy starts queue right away (AsIs), it's almost impossible, or at least really delayed to add another files after first group to finish the operation.

So such feature would really help in such cases.

Thank you in advance for considering this.

**Comment by avmaksimov** [comment]
I think better to make possibility to add to some queue operation like in Unreal Commander.
